lib/Extension/Bundle/Installer/SettingsStoreAwareInstaller.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+
+namespace Pimcore\Extension\Bundle\Installer;
+
+
+use Pimcore\Migrations\MigrationManager;
+use Pimcore\Model\Tool\SettingsStore;
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+
```

Comment:
Should we maybe call this class `SettingsStoreMigrationAwareInstaller` or `SettingsStoreAwareMigrationInstaller` ? Because it includes the migration support as well? 🤔wdyt?